bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java

```diff
@@ -431,9 +427,10 @@ private void scheduleAlertStateRestore(Command command) {
      * restoration.
      */
     private void cancelScheduledFuture() {
+        ScheduledFuture<?> scheduledFuture = this.scheduledFuture;
         if (scheduledFuture != null) {
             scheduledFuture.cancel(true);
-            scheduledFuture = null;
+            this.scheduledFuture = null;
```

Comment:
In my opinion renaming the local variable would be a better pattern instead of  using this operator?

Reply:
Done